vtexdocs/helpcenter
# Feat/start here

#### What is the purpose of this pull request?

To update the start here page to follow the high fidelity prototype. 

#### What problem is this solving?

This PR adds:
- Reading time
- Tracks enumeration (in the sidebar, breadcrumbs and documentation title)
- I kept the current version of the table of contents, contributors and suggest edits components.]

More information: [Notion](https://www.notion.so/vtexhandbook/Telas-Comece-Aqui-b101c3b6ad12484093185e5b62baa4e1?pvs=4#3776217d3a5347a8adc22900f1f00e5c)

#### How should this be manually tested?

Open the start here page, check if the [start here page](https://deploy-preview-23--leafy-mooncake-7c2e5e.netlify.app/pt/docs/tracks) contains all of the required cards (compare with the [current Help Center](https://help.vtex.com/tracks)) and navigate across the documentation pages. Keep in mind that some documentation pages are broken and are not working.

**_⚠️ The enumeration in the sidebar will not appear in the preview, because it depends on the changes made to the navigation.json in this PR and the json is fetched from the main branch version. So you would need to test it locally, fetching from the updated navigation. ⚠️_**

#### Screenshots or example usage

In red the components that were created or updated, in green components that already existed and were kept in the documentation page even if they were not shown in the prototype.

![image](https://github.com/vtexdocs/helpcenter/assets/62757720/39e59f5b-2562-463b-a259-1b647a734293)
![image](https://github.com/vtexdocs/helpcenter/assets/62757720/bc1ddd5d-896c-4540-9edf-f9a922cd107a)


#### Types of changes
